### PR TITLE
tests: uprev miniconda

### PR DESCRIPTION
**Pull request by sergiusens**
Newer version required to handle internal changes in anaconda/conda.

Signed-off-by: Sergio Schvezov <sergio.schvezov@canonical.com>

- [ ] Have you followed the [guidelines for contributing](https://github.com/snapcore/snapcraft/blob/master/CONTRIBUTING.md)?
- [ ] Have you signed the [CLA](http://www.ubuntu.com/legal/contributors/)?
- [ ] Have you successfully run `make lint`?
- [ ] Have you successfully run `pytest tests/unit`?

-----

### CODE REVIEW

**Comment by cmatsuoka** [comment]
Maybe version `py37_4.12.0` for plugins v1 and `py39_4.12.0` for the craft-parts plugin?

File: tests/spread/plugins/v2/snaps/conda-hello/snap/snapcraft.yaml [hunk 1]
```diff
@@ -16,7 +16,7 @@ apps:
 parts:
   ipython:
     plugin: conda
-    conda-miniconda-version: 4.6.14
+    conda-miniconda-version: "py39_4.12.0"
```
Comment (cmatsuoka) [inline tests/spread/plugins/v2/snaps/conda-hello/snap/snapcraft.yaml:19]: This version is mismatched with the conda-python-version.

**Review by cmatsuoka — APPROVED** [review]
It looks fine, there's a version mismatch but if tests pass it should be ok.

File: tests/spread/plugins/v2/snaps/conda-hello/snap/snapcraft.yaml (hunk 1 above)
Comment (sergiusens) [reply tests/spread/plugins/v2/snaps/conda-hello/snap/snapcraft.yaml:19]: made 3.9 match too

**Pull request merged**